src/v2i-hub/ImmediateForwardPlugin/manifest.json

```diff
@@ -15,7 +15,7 @@
     },
     {
       "key": "ImmediateForwardConfigurations",
-      "default": "[{\"name\": \"East Intersection Cohda\", \"rsuSpec\": \"RSU4.1\",\"address\": \"127.0.0.1\",\"port\": 1516,\"txMode\": \"CONT\",\"signMessages\": false,\"messages\": [{ \"tmxType\": \"SPAT-P\", \"sendType\": \"SPAT\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"MAP-P\", \"sendType\": \"MAP\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"PSM-P\", \"sendType\": \"PSM\", \"psid\": \"0x27\", \"channel\": 183 }, { \"tmxType\": \"TIM\", \"sendType\": \"TIM\", \"psid\": \"0x8003\", \"channel\": 183 },{ \"tmxType\": \"TMSG07-P\", \"sendType\": \"TMSG07\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"TMSG03-P\", \"sendType\": \"TMSG03\", \"psid\": \"0xBFEE\", \"channel\": 183 },{ \"tmxType\": \"TMSG05-P\", \"sendType\": \"TMSG05\", \"psid\": \"0x8003\", \"channel\": 183 },{ \"tmxType\": \"SSM-P\", \"sendType\": \"SSM\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"SDSM\", \"sendType\": \"SDSM\", \"psid\": \"0x8010\", \"channel\": 183 }]}]",
+      "default": "[{\"name\": \"East Intersection Cohda\", \"rsuSpec\": \"RSU4.1\",\"address\": \"127.0.0.1\",\"port\": 1516,\"txMode\": \"CONT\",\"signMessages\": false,\"messages\": [{ \"tmxType\": \"SPAT-P\", \"sendType\": \"SPAT\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"MAP-P\", \"sendType\": \"MAP\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"PSM-P\", \"sendType\": \"PSM\", \"psid\": \"0x27\", \"channel\": 183 }, { \"tmxType\": \"TIM\", \"sendType\": \"TIM\", \"psid\": \"0x8003\", \"channel\": 183 },{ \"tmxType\": \"TMSG07-P\", \"sendType\": \"TMSG07\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"TMSG03-P\", \"sendType\": \"TMSG03\", \"psid\": \"0xBFEE\", \"channel\": 183 },{ \"tmxType\": \"TMSG05-P\", \"sendType\": \"TMSG05\", \"psid\": \"0x8003\", \"channel\": 183 },{ \"tmxType\": \"SSM-P\", \"sendType\": \"SSM\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"SDSM\", \"sendType\": \"SDSM\", \"psid\": \"0x8010\", \"channel\": 183 }]},{\"name\":\"Yunnex\",\"rsuSpec\":\"NTCIP1218\",\"address\":\"192.168.55.20\",\"port\":161,\"snmpAuth\":{\"user\":\"rsuRwUser\",\"securityLevel\":\"authPriv\",\"community\":\"public\",\"authProtocol\":\"SHA-512\",\"authPassPhrase\":\"St0lcav1!\",\"privacyProtocol\":\"AES-256\",\"privacyPassPhrase\":\"St0lcav1!\"},\"txMode\":\"CONT\",\"signMessages\":false,\"messages\":[{\"tmxType\":\"MAP-P\",\"sendType\":\"MAP\",\"psid\":\"0x0020\",\"channel\":183},{\"tmxType\":\"SPAT-P\",\"sendType\":\"SPAT\",\"psid\":\"0x8002\",\"channel\":183},{\"tmxType\":\"PSM-P\",\"sendType\":\"PSM\",\"psid\":\"0x0027\",\"channel\":183},{\"tmxType\":\"TIM\",\"sendType\":\"TIM\",\"psid\":\"0x8003\",\"channel\":183},{\"tmxType\":\"SSM-P\",\"sendType\":\"SSM\",\"psid\":\"0x8002\",\"channel\":183},{\"tmxType\":\"SDSM\",\"sendType\":\"SDSM\",\"psid\":\"0x8010\",\"channel\":183}]}]",
```

Comment:
Can you update the MAP PSID to either 8002 or the new E0000017?
https://standards.ieee.org/products-programs/regauth/psid/public/
"MAP distribution"